inference-engine/src/mkldnn_plugin/nodes/mkldnn_scatter_update_node.cpp

```diff
@@ -167,7 +167,8 @@ void MKLDNNScatterUpdateNode::initSupportedPrimitiveDescriptors() {
     auto dataType = MKLDNNExtensionUtils::IEPrecisionToDataType(dataPrec);
     dataSize = MKLDNNExtensionUtils::sizeOfDataType(dataType);
 
-    bool canBeInplace = getParentEdgeAt(DATA_ID)->getParent()->getChildEdges().size() == 1;
+    bool canBeInplace = getParentEdgeAt(DATA_ID)->getParent()->getChildEdges().size() == 1 &&
+            !getParentEdgeAt(0)->getParent()->isConstant();
```

Comment:
maybe better to use DATA_ID as well:
`!getParentEdgeAt(DATA_ID)->getParent()->isConstant()`
then if input port order need change, just need change DATA_ID

Reply:
Done